en/components/grid.md

```diff
@@ -639,3 +638,17 @@ Our community is active and always welcoming to new ideas.
 
 * [Ignite UI for Angular **Forums**](https://www.infragistics.com/community/forums/f/ignite-ui-for-angular)
 * [Ignite UI for Angular **GitHub**](https://github.com/IgniteUI/igniteui-angular)
+
+## Grid Limitations
+
+<div ckass="devider"></div>
+
+### General
+
+|Limitation|Description|
+|--- |--- |--- |
+|Column widths into `percentage`|Currently we do not support mixing column widths with `%` and `px`.
+|Skipped `height` on the parent of the grid or grid itself| The `grid` is not rendered correctly whenever the `height` on the parent or on the grid itself is omitted.|
+|`NaN` whenever filter number column|When filter number column and enter character different from digit, `NaN` value appers due incorrect cast and no filter is applied.|
```

Comment:
`NaN` is returned when filtering columns of type number | If value type different than number is entered in the filtering input, `NaN` value appers due incorrect cast and no filter is applied.